backend/prisma/schema.prisma

```diff
@@ -184,13 +185,12 @@ model Thread {
 }
 
 model Image {
-  id                 Int       @default(autoincrement()) @id
-  smallSize          String
-  largeSize          String
-  postId             String?
-  imageRole          ImageRole
-  Post               Post?     @relation(fields: [postId_ImageToPost], references: [id])
-  postId_ImageToPost Int?
+  id        Int       @default(autoincrement()) @id
+  smallSize String
+  largeSize String
+  imageRole ImageRole
+  post      Post      @relation(fields: [postId], references: [id])
+  postId    Int
```

Comment:
@Lanny it feels like it should have been like this, right? I think Prisma autogenerated that previous one perhaps due to us not switching the previous `postId` from `String` to `Int` perhaps?

Reply:
Yeah, posts have integer IDs so `postId` should definitely be an int. The string type was probably a holdover from uuids like you said